shopfloor_reception_mobile/static/src/scenario/reception_states.js

```diff
@@ -131,11 +131,14 @@ export const reception_states = function () {
             },
             on_scan: (barcode) => {
                 // Scan a lot
+                const input = document.querySelector('input[name="expiration_date"]');
```

Comment:
You should hook to event provided by the widget instead and collect this info somewhere else (see `on_date_picker_selected`). As an alternative you can declare a `$ref` on the widget where is used.